clippy_lints/src/lib.rs

```diff
@@ -924,6 +925,11 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
         ))
     });
     store.register_late_pass(|_| Box::new(no_mangle_with_rust_abi::NoMangleWithRustAbi));
+    store.register_late_pass(move |_| {
+        Box::new(unnecessary_box_returns::UnnecessaryBoxReturns::new(
+            avoid_breaking_exported_api,
```

Comment:
Could you also add the lint to the configuration docs in `clippy_lints::utils::conf`? Running `cargo collect-metadata` should then update the documentation automatically